components/banner/styles.ts

```diff
@@ -0,0 +1,34 @@
+import styled, { css, CssWithTheme } from "styled-components";
+
+export const Section = styled.section`
+  color: #fff;
+  background: ${({ theme }) => theme.colors.darkGrey};
+  margin-bottom: 20px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.lg}) {
+    margin-bottom: 0;
+  }
+`;
+
+export const Container = styled.div`
+  max-width: ${({ theme }) => theme.contentSize}px;
+  padding: 40px 20px 40px 20px;
+  margin: auto;
+  position: relative;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.lg}) {
+    padding: 10px 20px 20px 20px;
+  }
+`;
+
+export const Content = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: ${({ theme }) => `${theme.space.md}px ${theme.space.md}px`};
+  position: relative;
+  width: 100%;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.lg}) {
```

Comment:
Muzeme to dat trosku nize? At je to na tabletech vedle sebe. Ten roztazeny widget neni moc hezky.

Reply:
To je dobrý návrh, díky! Předělám a force-pushnu.

Reply:
Opraveno v bc3dba2.